projects/observability/src/shared/dashboard/data/graphql/explore/explore-cartesian-data-source.model.ts

```diff
@@ -145,7 +145,12 @@ export abstract class ExploreCartesianDataSourceModel extends GraphQlDataSourceM
                 attribute: request.attributes?.find(
                   attribute => attribute.name === request.groupBy?.keyExpressions?.[0].key
                 ),
-                metadata: request.attributes
+                metadata: request.attributes,
```

Comment:
Pre-existing, but if we're picking out the appropriate attribute why do we need to send the whole list down too? Also attribute and metadata are basically synonyms in our system - as can be seen here since they have the same datatype. We should be using consistent names, especially in the same type (e.g. targetAttribute and allAttributes).

Reply:
allAttributes necessary to redirect on filter click (from filter-url.service), anyway renaming them

Reply:
Setting the url (going from object to string) shouldn't require all attributes; reading it does (to go from string to object)

Reply:
yes, you're right. let me remove it